### ISSUE TITLE
why   not   change  from “tf.nn. fused_batch_norm”  to  "tf.nn.batch_normalization"?  

### ISSUE THREAD

**Issue by sudongxiang** [jianlong-yuan/syncbn-tensorflow#6]
First  of all, well done, buddy!   
when I ran  this  program,I met  the  problem  "rank  not  match" at the  place  fused_batch_norm,  so  I   replace   “fused_batch_norm”  to  “batch_normalization” ， and  it  worked~

**Comment by jianlong-yuan** [comment]
Yes, it worked. The accuracy of the results is slightly different.
